tests/testsuite/rename_deps.rs

```diff
@@ -27,7 +27,7 @@ fn rename_dependency() {
         .file("src/lib.rs", "extern crate bar; extern crate baz;")
         .build();
 
-    p.cargo("build").run();
+    p.cargo("check").run();
```

Comment:
Since these involve making sure that the renamed libraries can be linked, I would probably avoid any changes to `rename_deps.rs`.